Duplicati/UnitTest/CommandLineOperationsTests.cs

```diff
@@ -52,13 +57,49 @@ orderby x
         public override void OneTimeSetUp()
         {
             base.OneTimeSetUp();
-            
-            using (WebClient client = new WebClient())
+
+            if (!File.Exists($@"{BASEFOLDER}\{zipAlternativeFilename}"))

```

Comment:
Can we use `Path.Combine` here instead of `\` here?

Reply:
Fixed. Thanks.